examples/simple_external_control_server/Readme.md

```diff
@@ -0,0 +1,12 @@
+# Simple External Control Server example
+This simple server, serves the External Control with URScript provided by a file
+
+Example of use:
+```bash
+python3 simple_external_control_server.py -p 50002 hello_world.script
```

Comment:
We could omit the port parameter in the example execution, as it does have a default value. However, it also does not hurt to specify it, so feel free to ignore this comment.